poker-hand-evaluator/src/Main.java

```diff
@@ -0,0 +1,106 @@
+import evaluator.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    static DeckOfCards deck = new DeckOfCards();
+    static List<Hand> winnerHand = new ArrayList<>();
+
+    public static void main(String[] args) throws InvalidHandSizeException {
+
+        Player player1 = new Player("Player_A");
+        Player player2 = new Player("Player_B");
+
+        startOfDealing(player1);
+        startOfDealing(player2);
+
+        flopOfDealing(player1);
+        flopOfDealing(player2);
+
+        turnOfDealing(player1);
+        turnOfDealing(player2);
+
+        riverOfDealing(player1);
+        riverOfDealing(player2);
```

Comment:
The flop, the turn and the river is the same card for each player. So to sum up:

- All players have 2 cards in hand at the beginning
- Flop: 3 cards are put on the table
- Turn: 1 card is put on the table
- River: 1 card is put on the table
Everyone has two individual cards, plus the flop, turn and river, which are the same cards for every player at the table!